README.md

```diff
@@ -80,38 +743,419 @@ wp theme
     | version | 1.2            |
     +---------+----------------+
 
-    # Get status of theme
-    $ wp theme status twentysixteen
-    Theme twentysixteen details:
-    		Name: Twenty Sixteen
-    		Status: Active
-    		Version: 1.2
-    		Author: the WordPress team
 
 
+### wp theme install
+
+Install a theme.
+
+~~~
+wp theme install <theme|zip|url>... [--version=<version>] [--force] [--activate]
+~~~
+
+**OPTIONS**
+
+	<theme|zip|url>...
+		A theme slug, the path to a local zip file, or URL to a remote zip file.
+
+	[--version=<version>]
+		If set, get that particular version from wordpress.org, instead of the
+		stable version.
+
+	[--force]
+		If set, the command will overwrite any installed version of the theme, without prompting
+		for confirmation.
+
+	[--activate]
+		If set, the theme will be activated immediately after install.
+
+**EXAMPLES**
+
+    # Install the latest version from wordpress.org and activate
+    $ wp theme install twentysixteen --activate
+    Installing Twenty Sixteen (1.2)
+    Downloading install package from http://downloads.wordpress.org/theme/twentysixteen.1.2.zip...
+    Unpacking the package...
+    Installing the theme...
+    Theme installed successfully.
+    Activating 'twentysixteen'...
+    Success: Switched to 'Twenty Sixteen' theme.
+
+    # Install from a local zip file
+    $ wp theme install ../my-theme.zip
+
+    # Install from a remote zip file
+    $ wp theme install http://s3.amazonaws.com/bucketname/my-theme.zip?AWSAccessKeyId=123&Expires=456&Signature=abcdef
+
 
-### wp theme mod
 
-Manage theme mods.
+### wp theme is-installed
+
+Check if the theme is installed.
 
 ~~~
-wp theme mod
+wp theme is-installed <theme>
 ~~~
 
+Returns exit code 0 when installed, 1 when uninstalled.
+
+**OPTIONS**
+
+	<theme>
+		The theme to check.
+
 **EXAMPLES**
 
-    # Set the 'background_color' theme mod to '000000'.
-    $ wp theme mod set background_color 000000
-    Success: Theme mod background_color set to 000000
+    # Check whether theme is installed; exit status 0 if installed, otherwise 1
+    $ wp theme is-installed hello
+    $ echo $?
+    1
+
+
+
+### wp theme list
+
+Get a list of themes.
+
+~~~
+wp theme list [--<field>=<value>] [--field=<field>] [--fields=<fields>] [--format=<format>]
+~~~
+
+**OPTIONS**
+
+	[--<field>=<value>]
+		Filter results based on the value of a field.
+
+	[--field=<field>]
+		Prints the value of a single field for each theme.
+
+	[--fields=<fields>]
+		Limit the output to specific object fields.
+
+	[--format=<format>]
+		Render output in a particular format.
+		---
+		default: table
+		options:
+		  - table
+		  - csv
+		  - json
+		  - count
+		  - yaml
+		---
+
+**AVAILABLE FIELDS**
+
+These fields will be displayed by default for each theme:
+
+* name
+* status
+* update
+* version
+
```

Comment:
Mention that the status field can have values "active", "inactive", "parent".
Mention that the update field can have values "available", "none".